client/src/app/components/CreateProject/Options/Option.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./options.css";
+const classNames = require("classnames");
+
+class Option extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isActive: this.props.isActive };
+  }
+
+  componentDidMount() {
+    if (this.props.isDefault === this.props.name) {
+      this.setState({ isActive: true });
+      this.props.checkMax(1, this.props.name);
+    }
+  }
+
+  componentWillUpdate(props) {
+    if (props.reset !== this.props.reset) {
+      if (this.props.reset) {
+        this.setState({ isActive: false });
+      }
+    }
+  }
+
+  handleClick() {
+    if (this.state.isActive) {
```

Comment:
Same here, you would benefit from destructuring state and props variables on top